modular_skyrat/modules/aesthetics/flag/code/signs_flags.dm

```diff
@@ -0,0 +1,158 @@
+/obj/structure/sign/flag
+	name = "blank flag"
+	desc = "The flag of nothing. It has nothing on it. Magnificient."
+	icon = 'modular_skyrat/modules/aesthetics/flag/icons/flags.dmi'
+	icon_state = "flag_coder"
+	buildable_sign = FALSE
+	custom_materials = null
+	var/item_flag = /obj/item/sign/flag
+
+/obj/structure/sign/flag/wrench_act(mob/living/user, obj/item/wrench/I)
+	return
+
+/obj/structure/sign/flag/welder_act(mob/living/user, obj/item/I)
+	return
+
+/obj/item/sign/flag/welder_act(mob/living/user, obj/item/I)
+	return
+
+/obj/structure/sign/flag/MouseDrop(over_object, src_location, over_location)
+	. = ..()
+	if(over_object == usr && Adjacent(usr))
+		if(!item_flag || src.flags_1 & NODECONSTRUCT_1)
+			return
+		if(!usr.canUseTopic(src, BE_CLOSE, NO_DEXTERITY, FALSE, TRUE))
+			return
+		usr.visible_message(span_notice("[usr] grabs and folds \the [src.name]."), span_notice("You grab and fold \the [src.name]."))
+		var/obj/item/C = new item_flag(loc)
+		TransferComponents(C)
+		usr.put_in_hands(C)
```

Comment:
Single-letter variable bad.
```suggestion
		var/obj/item/flag_item = new item_flag(loc)
		TransferComponents(flag_item)
		usr.put_in_hands(flag_item)
```

Reply:
Copied from the original sign code, but I agree.